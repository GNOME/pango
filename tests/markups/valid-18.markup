<span font="Sans Italic 12"
      foreground="#00ff00"
      background="#f0f0f0"
      underline="double"
      underline_color="red"
      strikethrough="y"
      strikethrough_color="blue"
      fallback="n"
      lang="de"
      gravity="south"
      gravity_hint="strong">Lots of attrs</span>
