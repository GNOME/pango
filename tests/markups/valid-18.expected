Lots of attrs


---

range 0 13
[0,13]font-desc=Sans Italic 12
[0,13]foreground=#0000ffff0000
[0,13]background=#f0f0f0f0f0f0
[0,13]underline=2
[0,13]underline-color=#ffff00000000
[0,13]gravity=0
[0,13]gravity-hint=1
[0,13]strikethrough=1
[0,13]strikethrough-color=#00000000ffff
[0,13]fallback=0
[0,13]language=de
range 13 2147483647


---

[0:13] de Sans Italic 12
[13:2147483647] (null) Sans Italic 12
